gitprovider.go

```diff
@@ -102,3 +78,16 @@ type InternalHolder struct {
 func (ih InternalHolder) GetInternal() interface{} {
 	return ih.Internal
 }
+
+// Equals returns true if both a and b marshalled to JSON equal
+func Equals(a, b interface{}) (bool, error) {
```

Comment:
Call this something like `JSONEquals` to provide the context that you need to pass tagged structs?

Reply:
obsolete now